src/ui/atoms/tooltip/usage.mdx

```diff
@@ -5,40 +5,68 @@ package: 'woly'
 ---
 
 import {block, Playground} from 'lib/playground'
-import {Button, Input, Tooltip} from 'ui'
+import {Button, Input, Tooltip, Text} from 'ui'
+import {IconInfo} from 'static/icons';
 
-`Tooltip` is used when you want to show addition information near component.
-Tooltip message is visible when user hover over component that is passed as a children to `Tooltip`.
+`Tooltip` is used to show addition information near component. The information should be contextual, useful, and nonessential.
+
+A tooltip can be simple message box that is displayed when a user hovers over, clicks or gives focus to a UI element
+such as an icon, a highlighted word, or a button that is passed as a children to `Tooltip`.
+Also the definition tooltip provides additional help or defines an item or term. It may be used on the label of a UI element,
+or on a word embedded in a paragraph.
+
+Interactive tooltips may contain rich text and other interactive elements like buttons or links.
+In general, hiding interactive content in a tooltip is discouraged.
+Interactive tooltips are best used for onboarding experiences and product tours.
+
+The tooltip position is flexible and changes depending on how close the element is to the edge of the screen.
+The top, right, bottom, and left positions appear outside the target element, center-aligned along the appropriate axis.
```

Comment:
```suggestion
`Tooltip` is used to show additional information near the component. The information should be contextual, useful, and nonessential.

A tooltip can be a simple message box that is displayed when a user hovers over, clicks, or gives focus to a UI element
such as an icon, a highlighted word, or a button that is passed as children to `Tooltip`.
Also, the definition tooltip provides additional help or defines an item or term. It may be used on the label of a UI element,
or on a word embedded in a paragraph.

Interactive tooltips may contain rich text and other interactive elements like buttons or links.
In general, hiding interactive content in a tooltip is discouraged.
Interactive tooltips are best used for onboarding experiences and product tours.

The tooltip position is flexible and changes depending on how close the element is to the edge of the screen.
The top, right, bottom, and left positions appear outside the target element, center-aligned along the appropriate axis.
```